storage/pebble/lookup.go

```diff
@@ -14,13 +14,38 @@ type lookupKey struct {
 	encoded []byte
 }
 
+func newHeightKey(identifier byte) []byte {
+	key := make([]byte, 0, MinLookupKeyLen)
+	key = append(key, identifier)
+	key = append(key, '/')
+	key = binary.BigEndian.AppendUint64(key, placeHolderHeight)
+	return key
+}
+
+// LatestHeightKey is a special case of a lookupKey
+// with keyLatestBlockHeight as key, no owner and a placeholder height of 0.
+// This is to ensure SeekPrefixGE in pebble does not break
+func latestHeightKey() []byte {
```

Comment:
can we make it as a const or a variable

Reply:
You can refer to how defaultHashes is initialized here
https://github.com/onflow/flow-go/blob/master/ledger/trie.go#L51